pkgs/by-name/in/infrastructure-agent/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "infrastructure-agent";
+  version = "1.57.2";
+
+  src = fetchFromGitHub {
+    owner = "newrelic";
+    repo = "infrastructure-agent";
+    rev = version;
+    hash = "sha256-74aBFGBrrEiqAVG3kVYLOz3P4VTCw4G3vV/iBucFPYs=";
+  };
+
+  vendorHash = "sha256-0WLL15CXRi/flp4EV3Qt0wO1VaUmAokzsChpiqjs+YQ=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.buildVersion=${version}"
+    "-X main.gitCommit=${src.rev}"
+  ];
+
+  CGO_ENABLED = if stdenv.isDarwin then "1" else "0";
+
+  subPackages = [
+    "cmd/newrelic-infra"
+    "cmd/newrelic-infra-ctl"
+    "cmd/newrelic-infra-service"
+  ];
+
+  meta = with lib; {
```

Comment:
Please avoid `with lib`.

See: https://nix.dev/guides/best-practices.html#with-scopes

Reply:
Thanks! Done for the `meta` attrset, but still using it for `maintainers` as more could be added to the list. Let me know if it's good practice to still not use it e.g. when the lists have few items.